src/docfx/lib/schema/JsonSchemaValidator.cs

```diff
@@ -155,6 +155,22 @@ private void ValidateObject(JsonSchema schema, List<Error> errors, JObject map)
                 }
             }
 
+            foreach (var keys in schema.Either)
```

Comment:
Can you help do a simple refactor to extract these blocks to `ValidateEither`, `ValidateAdditionalProperties`, etc...

Reply:
Sure, I will do this